server/db/seeds.js

```diff
@@ -50,8 +50,11 @@ function generateOodlets(quantity, oodlers, quantifiedThingies) {
   console.log(`Generating ${quantity} Oodlets...`);
 
   for (let i = 0; i < quantity; i++) {
+    let randCreateDate = faker.date.past();
```

Comment:
you mean 
let notSoRandCreateDate = moment().subtract(2 * i, 'weeks').toDate()
2*quantity is always same value :smile: